Lib/test/test_codecs.py

```diff
@@ -3415,5 +3415,19 @@ def test_rot13_func(self):
             'To be, or not to be, that is the question')
 
 
+class CodecNameNormalizationTest(unittest.TestCase):
+    """Test the normalizestring function via codecs module"""
+    def test_normalized_encoding(self):
```

Comment:
Can you use local variables for readability?

```
FOUND = (1, 2, 3, 4)
NOT_FOUND = (None, None, None, None)
```

Then replace the tuples with these variables.

Reply:
Ok, no problem. I will update it.